rbc/omniscidb.py

```diff
@@ -281,6 +281,40 @@ def __init__(self,
         # An user-defined device-LLVM IR mapping.
         self.user_defined_llvm_ir = {}
 
+    def __call__(self, *args, **kwargs):
+        msg = """Decorated functions called directly. Decorated function are
+        not meant to be called directly, but are registered and later
+        executed directly on the database.
+
```

Comment:
Just a suggestion, but what about the following?
```
Cannot call functions decorated by `RemoteOmnisci`.  These functions are registered on the database and can be called only from a SQL query.
```

Reply:
Sure, thanks for the suggestion.